make/common/Modules.gmk

```diff
@@ -310,14 +310,21 @@ MODULE_DEPS_MAKEFILE := $(MAKESUPPORT_OUTPUTDIR)/module-deps.gmk
 
 MODULE_INFOS := $(call FindAllModuleInfos, *)
 
+ifeq ($(TOOLCHAIN_TYPE),gcc)
+  CPP_FLAGS += -P - <
+else ifeq ($(TOOLCHAIN_TYPE), microsoft)
+  CPP_FLAGS += -EP -nologo
+endif
```

Comment:
Modules.gmk is included many places: I don't think we should be modifying 'CPP_FLAGS' please invent a new macro to capture this difference. Also it needs to work if TOOLCHAIN_TYPE is neither gcc nor microsoft.

Reply:
@keithc-ca , fixed (might need to be revisited for other toolchains, e.g xlc)

Reply:
@keithc-ca , @jdekonin  re-run build for fix, ready for review